es/cheatsheet-machine-learning-tips-and-tricks.md

```diff
@@ -1,257 +1,257 @@
 
 <br>
 
 **14. [Metric, Formula, Equivalent]**
 
-&#10230;
+&#10230; [Métrica, Fórmula, Interpretación]
 
 <br>
 
 **15. AUC ― The area under the receiving operating curve, also noted AUC or AUROC, is the area below the ROC as shown in the following figure:**
 
-&#10230;
+&#10230; AUC ― El área bajo la curva Característica Operativa del Receptor, también conocida como AUC o AUROC, es el área debajo del ROC, como se muestra en la siguiente figura:
 
 <br>
 
 **16. [Actual, Predicted]**
 
-&#10230;
+&#10230; [Actual, predicha]
 
 <br>
 
 **17. Basic metrics ― Given a regression model f, the following metrics are commonly used to assess the performance of the model:**
 
-&#10230;
+&#10230; Métricas básicas - dado un modelo de regresión f, las siguientes métricas se usan comúnmente para evaluar el rendimiento del modelo:
 
 <br>
 
 **18. [Total sum of squares, Explained sum of squares, Residual sum of squares]**
 
-&#10230;
+&#10230; [Suma total de cuadrados, suma de cuadrados explicada, suma residual de cuadrados]
 
 <br>
 
 **19. Coefficient of determination ― The coefficient of determination, often noted R2 or r2, provides a measure of how well the observed outcomes are replicated by the model and is defined as follows:**
 
-&#10230;
+&#10230; Coeficiente de determinación: el coeficiente de determinación, a menudo indicado como R2 o r2, proporciona una medida de lo bien que los resultados observados son replicados por el modelo y se define de la siguiente manera:
 
 <br>
 
 **20. Main metrics ― The following metrics are commonly used to assess the performance of regression models, by taking into account the number of variables n that they take into consideration:**
 
-&#10230;
+&#10230; Métricas principales: las siguientes métricas se utilizan comúnmente para evaluar el rendimiento de los modelos de regresión, teniendo en cuenta la cantidad de variables n que tienen en cuenta:
 
 <br>
 
 **21. where L is the likelihood and ˆσ2 is an estimate of the variance associated with each response.**
 
-&#10230;
+&#10230; donde L es la probabilidad y ˆσ2 es una estimación de la varianza asociada con cada respuesta.
 
 <br>
 
 **22. Model selection**
 
-&#10230;
+&#10230; Selección de modelo
 
 <br>
 
 **23. Vocabulary ― When selecting a model, we distinguish 3 different parts of the data that we have as follows:**
 
-&#10230;
+&#10230; Vocabulario - al seleccionar un modelo, distinguimos 3 partes diferentes de los datos que tenemos de la siguiente manera:
 
 <br>
 
 **24. [Training set, Validation set, Testing set]**
 
-&#10230;
+&#10230; [Conjunto de entrenamiento, Conjunto de validación, Conjunto de prueba]
 
 <br>
 
 **25. [Model is trained, Model is assessed, Model gives predictions]**
 
-&#10230;
+&#10230; [Modelo es entrenado, modelo es evaluado, modelo da predicciones]
```

Comment:
[Modelo es entrenado, Modelo es evaluado, Modelo da predicciones]
Just change the first letter to uppercase